src/CheckoutSdk/Marketplace/Payout/Response/Util/GetScheduleResponseTypeConverter.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Checkout.Marketplace.Payout.Response.Util
+{
+    public class GetScheduleResponseTypeConverter : JsonConverter
+    {
+        public override bool CanWrite => false;
+
+        public override bool CanConvert(Type objectType)
+        {
+            return typeof(GetScheduleResponse).GetTypeInfo().IsAssignableFrom(objectType.GetTypeInfo());
+        }
+
+        public override void WriteJson(
+            JsonWriter writer,
+            object value,
+            Newtonsoft.Json.JsonSerializer serializer)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override object ReadJson(
+            JsonReader reader,
+            Type objectType,
+            object existingValue,
+            Newtonsoft.Json.JsonSerializer serializer)
+        {
+            JObject jObject = JObject.Load(reader);
+            GetScheduleResponse response = new GetScheduleResponse();
+            foreach (var node in jObject)
+            {
+                if (Enum.TryParse(node.Key, out Currency currency) && node.Value != null)
```

Comment:
The order here should be the opposite, right? I mean `node.Value != null && Enum.TryParse(node.Key, out Currency currency)`